packages/expo-module-scripts/tsconfig.base.json

```diff
@@ -5,10 +5,11 @@
     "jsx": "react",
     "moduleResolution": "node",
     "esModuleInterop": true,
-    "types": ["jest", "jest-require"],
+    "types": ["node", "jest", "jest-require"],
```

Comment:
Same here, this addition seems like the prevention for some kind of error, but without a context I'm not sure if adding `node` is a correct way to resolve it.

Reply:
I added it because of `global`